src/redux/reducers/index.ts

```diff
@@ -5,13 +5,16 @@ import * as currentProject from "./currentProjectReducer";
 import * as recentProjects from "./recentProjectsReducer";
 import * as appError from "./appErrorReducer";
 import * as auth from "./authReducer";
+import * as tracks from "./trackReducer";
```

Comment:
I'm not sure about naming for `tracks` 🚚 🚚 🚚  How about `tracking`?

Reply:
Okay, it fits 😄